Project structure reorganization

s opened the pull request:
Hello,
This pull request has a project structure reorganization to allow developers find files easier and have a consistent file structure between Xcode folders and Finder. I have used Xcode9's sync feature to have this result.
I have also deleted two unused files as we can always get them back from history if needed.
I would like to hear what do you think about this change and feedbacks from you.

Thank you.

jeroenleenarts commented:
Hello Said, will check this out later. Do not worry. It just might take a while. :)

One thing I noticed,  you checked in the resulting Acknowledgement file generated by CocoaPods which is then copied to the Settings bundle. Can you make sure that file is not committed?

Also check the `.gitignore` file to see if there are any files/paths that need to be updated.

s commented:
Hello Jeroen,
Sorry for Acknowledgements.plist. I removed it from git and updated the .gitignore file. 
Meanwhile I wonder why there are two .gitignore files in parent and child directory.
Thanks!
